src/annotations/components/annotationForm/AnnotationStartTimeInput.tsx

```diff
@@ -30,6 +45,24 @@ export const AnnotationStartTimeInput: FC<Props> = (props: Props) => {
     }
   }
 
+  const isInputValueValid = (inputValue: string): boolean => {
```

Comment:
small nit, `isValidTimeFormat` vs `isInputValueValid` are kind of in different voices - consistency makes it easier to follow along and ultimately understand. Small thing, but I think these tend to add up.